tools/scylla-sstable.cc

```diff
@@ -929,6 +932,7 @@ void validate_operation(schema_ptr schema, reader_permit permit, const std::vect
         writer.EndObject();
     }
     writer.EndStream();
+    fmt::print(std::cout, "{}", json_output_stream.str());
```

Comment:
```suggestion
    fmt::print(std::cout, "{}", json_output_stream.view());
```

it's free though.

Reply:
see also https://github.com/scylladb/scylladb/pull/16216#discussion_r1408979361 . to use `view()` when possible not only help with the performance, it can help the posterity. as i think, people tend to replicate the existing pattern.

Reply:
Done.